src/analyzer/analyzergain.h

```diff
@@ -15,18 +15,18 @@ class ReplayGain;
 
 class AnalyzerGain : public Analyzer {
```

Comment:
Multiple versions of beat detection go through the same class -- why do we need an analyzer separate to AnalyzerGain? Seems to me you'd eliminate some code duplication and not need the isDisabledOrLoadStoredSuccess API change.